ui/widgets/resizable.js

```diff
@@ -690,6 +699,45 @@ $.widget( "ui.resizable", $.ui.mouse, {
 		};
 	},
 
+	_calculateAdjustedElementDimensions: function( element ) {
+		var ce = element.get( 0 );
+
+		if ( element.css( "box-sizing" ) !== "content-box" ||
+			( !this._hasScroll( ce ) && !this._hasScroll( ce, "left" ) ) ) {
+				return {
+					height: parseFloat( element.css( "height" ) ),
+					width: parseFloat( element.css( "width" ) )
+				};
+		}
+
+		// Check if CSS inline styles are set and use those (usually from previous resizes)
+		var elWidth = ce.style.width === "" ? "" : parseFloat( ce.style.width );
+		var elHeight = ce.style.height === "" ? "" : parseFloat( ce.style.height );
+
+		if ( elWidth === "" ) {
+			elWidth = this._getElementSizeWithoutOverflow( element, "width" );
+		}
+		if ( elHeight === "" ) {
+			elHeight = this._getElementSizeWithoutOverflow( element, "height" );
+		}
+
+		return {
+			height: elHeight,
+			width: elWidth
+		};
+	},
+
+	_getElementSizeWithoutOverflow: function( element, sizeProperty ) {
+		var overflowProperty = sizeProperty === "width" ? "overflow-y" : "overflow-x";
+
+		var origOverflow = element.css( overflowProperty );
+		element.css( overflowProperty, "hidden" );
+		var elSize = parseFloat( element.css( sizeProperty ) );
```

Comment:
This will trigger an extra layout per function call which is undesirable. Since it gets called twice, it's two extra layouts, which is even worse.

There are other ways to get a scrollbar size, e.g. the hack that jQuery itself employs; see https://github.com/jquery/jquery/blob/dbc9dac7aecb106b66050342ff8daf1ecdd4239f/src/css.js#L99-L110

Reply:
Thanks for the suggestion!

I have changed the calculation to use something similar to the "jQuery hack" you linked.
In this case, I used it to calculate the "theoretical" size of the element instead of scrollbar size delta.